roles/network_plugin/kube-ovn/templates/cni-kube-ovn.yml.j2

```diff
@@ -580,7 +434,7 @@ spec:
         type: infra
     spec:
       priorityClassName: system-node-critical
-      serviceAccountName: ovn
+      serviceAccountName: kube-ovn-app
       hostPID: true
```

Comment:
Is this safe to upgrade this way on running cluster ? If the sa changes, I suppose that will leave a dangling SA, right ?

Reply:
Not tested the upgrade before, it should work. I test the deployment in the new env. actually it changed in the upsteam kube-ovn.

<img width="1653" alt="image" src="https://github.com/user-attachments/assets/c22e8e2f-0b13-44d4-99bf-bac57d518e91" />

ovn SA is still in use, so it is not dangling SA.

<img width="1449" alt="image" src="https://github.com/user-attachments/assets/d2393983-bd85-44ad-8c3b-facfeeb81a97" />

Reply:
Ah ok so it was using one before and now it's two ? Understood.

Reply:
> Ah ok so it was using one before and now it's two ? Understood.

yes